include/seqan3/alignment/scoring/scoring_scheme_concept.hpp

```diff
@@ -57,11 +57,14 @@ namespace seqan3
 //!\}
 
 //!\cond
-template <typename t, alphabet alphabet_t, alphabet alphabet2_t = alphabet_t>
+template <typename t, typename alphabet_t, typename alphabet2_t = alphabet_t>
 SEQAN3_CONCEPT scoring_scheme = requires (t scheme,
                                                 alphabet_t const alph1,
                                                 alphabet2_t const alph2)
```

Comment:
If you already touch this file, maybe align these two lines with the line above?

Reply:
done